QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+{
+	"@graph": [
+		{
+			"@id": "res:acqf",
+			"@type": "???:QualificationsFramework",
+			"ceasn:name": { "en": "African Continental Qualifications Framework" },
+			"ceterms:alternateName": { "en": "ACQF" },
+			"ceasn:description": { "en": "The ACQF is a comprehensive and inclusive meta-referencing qualifications framework, designed to support a holistic and systemic vision of learning, qualifications, and credentials. The ACQF is oriented to lifelong learning and supports parity of esteem of learning outcomes acquired in formal, non-formal and informal contexts, and all levels of education and training." },
+			"ceasn:dateValidFrom": "2022-11-01",
+			"ceterms:versionCode": "17",
```

Comment:
Cover page shows "Version 17 November 2022"---remember how the rest of the world does dates :-) 
p2, version control shows this as version 6 .  

I would record the former as date valid from, the latter as the versionCode.